analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.TryCatchFinally.cs

```diff
@@ -622,11 +617,8 @@ public void TryCatch_ThrowInCatch_SingleCatchBlock()
                 "InCatch",
                 "AfterCatch"); // If there is no exception in try
 
-            validator.TagStates("InCatch").Should().HaveCount(1)
-                     .And.ContainSingle(x => HasUnknownException(x));
-
-            validator.TagStates("AfterCatch").Should().HaveCount(1)
-                     .And.ContainSingle(x => HasNoException(x));
+            validator.TagStates("InCatch").Single().Should().HaveUnknownException();
+            validator.TagStates("AfterCatch").Single().Should().HaveNoException();
```

Comment:
If I break this to expect the unknown exception in both places:
_Before:_ It's clear what failed and why.
```
  Message: 
    Expected validator.TagStates("AfterCatch") to contain a single item matching HasUnknownException(x), but no such item was found.
```
_After:_ It is not clear what failed. only why.
```
Message: 
Expected program state to have an unknown exception but found <null>
```
The information about `validator.TagStates("AfterCatch")` is lost. Can you add it as a part of the message?